deploy/environments/applications.yaml

```diff
@@ -58,7 +58,7 @@ spack:
     - py-morphio
     - py-morphology-repair-workflow
     - py-morph-tool
-    - py-neurodamus
+    - py-neurodamus%gcc ^neuron%intel
```

Comment:
I think this is needed as in https://github.com/BlueBrain/spack/pull/1066, but please confirm if it's correct.

Reply:
yes - if a) package is compiled with non-intel compiler and 2) you are adding neuron as dependency

Reply:
As discussed on Slack, we should avoid neuron dependency on py-neurodamus.